shared/src/commonMain/kotlin/io/customer/shared/common/Extensions.kt

```diff
@@ -0,0 +1,7 @@
+package io.customer.shared.common
+
+/**
+ * Helper extension created for [QueueTaskResult] to make it easier to visualize success state.
+ */
+@Suppress("FunctionName")
+internal fun Boolean.Companion.Success() = true
```

Comment:
This is actually created for `QueueTaskResult` to give visual clarity for task results. Please see my comment on `QueueTaskResult` to see the idea behind.